.travis.yml

```diff
@@ -1,16 +1,27 @@
 language: go
 
+# cache:
+#   directories:
+#   - $HOME/.glide
+
 go:
 - 1.7.1
 
 branches:
   only:
   - master
   - cluster-up
+  - /^issue-[0-9]+$/
 
 install:
+# Work around the fact that I don't seem to be able to change the git checkout command
+# In order to build any fork, we need to make sure that the code lives in 'minishift'
 - mkdir /home/travis/gopath/bin
 - mkdir /home/travis/gopath/pkg
+- cd $HOME/gopath/src/github.com
+- mv `ls` foo
```

Comment:
The git clone command on a fork looks like this:

     $ git clone https://github.com/hferentschik/minishift.git hferentschik/minishift

Check for example my fork build - https://travis-ci.org/hferentschik/minishift

You are ending up with something like $GOPATH/src/github.com/hferentschik/minishift

Thanks to the way how go and imports work, this will in our case not even compile. It has to be $GOPATH/src/github.com/minishift/minishift. IMO this is plain out stupid, but it is the way Go works. Anyways, three options here:

* Find a way to change the clone command in Travis. So far I could not find anything. 
* Change to relative imports in the source code. TBH, this is something I have been toying with. This would solve this issue as well as the issue where we had to change all imports when we changes the repo name. I find it very odd that the source is tied to the repo. However, the recommended Go approach seems to be the fully qualified imports. It's on my todo list to look at this a bit more. For now not an option.
* Last but not least, you need a way to make the checkout match the expectation, aka rename the directory. I cannot just do 'mv `ls` minishift' since in the case of a main repo build this will create an error (I cannot move a directory onto itself). So in the end I rename the directory to whatever (foo) and then to minishift. This will work no matter on whether I am on the origin repo or a fork